apidoc/Titanium/UI/WebView.yml

```diff
@@ -497,6 +498,7 @@ events:
     summary: Fired when the web view content is loaded.
     properties:
       - name: url
+        type: String
         summary: URL of the web document.
```

Comment:
Ti.UI.WebView's `load` event has a property `title` on iOS, which is a `String` and is "The page title."